README.md

```diff
@@ -41,7 +52,11 @@ y = 10;
 
 After each line of code executes, what are the values of x and y?  Do the variables change?  Do their values?
 
-<!-- Replace this comment with your answer -->
+> The value of "y" is 10.
+> The value of "x" is the variable "y", which evaluates to 10.
```

Comment:
No, sorry. `x = y` does not form any sort of persistent relationship between `x` and `y`; instead, `y` is evaluated first, and the _value_ of `y` gets assigned to `x`.